blockstack/blockstackd.py

```diff
@@ -1423,7 +1423,7 @@ def export_account_state(self, account_state):
         """
         Make an account state presentable to external consumers
         """
```

Comment:
@jcnelson @kantai this location for a patch was not intuitive for me. There was another area that made more sense, but found this after some local testing. Please confirm

Reply:
I think @jcnelson has the most context on the appropriate place to apply this. Alternatively, you can catch the forwarding that happens in `api/server.py` and rewrite the lock time there.

Reply:
All this change will do is alter what is (a) returned by the RESTful API and (b) what is returned by `blockstack/lib/client.py`.  If that's what you want, then this is fine.  I think it's more appropriate to do it here than in `api/server.py`.